virtualization/install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+apt-get update && apt-get install -y libvirt-dev pkg-config
```

Comment:
The FAME user should not have apt-get privilege, so this should probably be inside a README or inside an init check instead.

Reply:
No, when initializing the module dependencies the scripts are run with root privileges so that e.g. system packages can be installed before running the modules. The privileges are dropped afterwards.

Reply:
As far as I know this is not the case and that is why the installation scripts never contain `apt-get` statements, even when they are necessary (see `https://github.com/certsocietegenerale/fame_modules/blob/master/antivirus/mail/install.py` for example)

Reply:
You are right, I was mistaken in regards of when admin permissions are available or not. In the case of Docker, my assumption is correct (the module installation scripts are run as root). But in the bare-metal case you are perfectly right, the installation scripts do not have root permissions. I will take care of this.

Reply:
For now, I decided to only call the `install.sh` scripts when FAME runs in Docker. I will extend all modules that have an `install.sh` file to also have an `install.py` file which checks whether or not the `install.sh` was executed (eg. by checking for certain files on the filesystem).